src/System.ValueTuple/tests/System.ValueTuple.Tests.csproj

```diff
@@ -17,6 +19,9 @@
     <Compile Include="ExtensionsTests.cs" />
     <Compile Include="ValueTupleTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp1.1'">
+    <TargetingPackReference Include="System.Private.CoreLib" />
```

Comment:
You shouldn't need to reference S.P.CoreLib from the test project. YOu should be getting the APIs from your updated System.Runtime.